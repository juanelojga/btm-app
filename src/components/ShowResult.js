import React from 'react'
import PropTypes from 'prop-types'

const ShowSubmittedValue = ({value}) => <div>Submitted Value: {value}</div>

ShowSubmittedValue.propTypes = {
  value: PropTypes.string,
}

export default ShowSubmittedValue
