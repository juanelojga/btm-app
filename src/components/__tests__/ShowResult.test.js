import React from 'react'
import {render} from 'react-testing-library'
import faker from 'faker'

import ShowResult from '../ShowResult'

test('snapshot empty value', () => {
  const {container} = render(<ShowResult />)
  expect(container).toMatchSnapshot()
})

test('snapshot', () => {
  const value = faker.hacker.noun()
  const {container} = render(<ShowResult value={value} />)
  expect(container).toMatchSnapshot()
})
