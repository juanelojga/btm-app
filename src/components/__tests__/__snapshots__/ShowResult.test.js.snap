// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <div>
    Submitted Value: 
    interface
  </div>
</div>
`;

exports[`snapshot empty value 1`] = `
<div>
  <div>
    Submitted Value: 
  </div>
</div>
`;
